python/ray/workflow/recovery.py

```diff
@@ -166,8 +167,17 @@ def _construct_resume_workflow_from_step(
 
 @ray.remote(num_returns=2)
 def _resume_workflow_step_executor(
-    workflow_id: str, step_id: "StepID", current_output: [ray.ObjectRef]
+    job_id: str,
+    workflow_id: str,
+    step_id: "StepID",
+    current_output: [ray.ObjectRef],
 ) -> Tuple[ray.ObjectRef, ray.ObjectRef]:
+    # Re-configure log file to send logs to correct driver.
+    node = ray.worker._global_node
+    out_file, err_file = node.get_log_file_handles(
+        get_worker_log_file_name("WORKER", job_id)
+    )
+    configure_log_file(out_file, err_file)
```

Comment:
To make things safe, let's set it back when this function is finished.

Reply:
My concern is that maybe in WorkflowManagerActor, it runs some remote tasks, and this worker is reused and this will make the log output to the wrong place.